pyhelm/chartbuilder.py

```diff
@@ -236,6 +233,18 @@ def get_helm_chart(self):
         self._helm_chart = helm_chart
         return helm_chart
 
+    @staticmethod
+    def read_file(self, path):
```

Comment:
When using staticmethods, there's no 'self' passed. Symmetrically, calls to this method are done not by referencing 'self', but by the class itself.

Reply:
Riiiight. I'll sort that. Apologies for having to school me here. I'm fairly new to Python

Reply:
New commits pushed

Reply:
No worries :)